Needs node-sass @ '>=3.0.0-pre <4.0.0-0' ?

`¯\_(ツ)_/¯`

The dump below is taken from Strider-CD when building from the last commit.

```
$ sh -x -c 'set -e npm install bower install ember build --environment production cp docker/Dockerfile dist cp bin/run.sh dist cp 200.jade dist cp -R public/assets dist cd dist COMMIT_HASH=$(git rev-parse HEAD) docker build -t quay.io/yldio/paz-web . docker tag quay.io/yldio/paz-web quay.io/yldio/paz-web:$COMMIT_HASH docker push quay.io/yldio/paz-web:latest docker push quay.io/yldio/paz-web:$COMMIT_HASH' custom 2m
+ set -e
+ npm install

> precommit-hook@1.0.7 install /home/ubuntu/.strider/data/yldio-paz-web-5526e7393fd72c3144000323/node_modules/precommit-hook
> hook_install


Creating .git/hooks/pre-commit...
npm ERR! notarget No compatible version found: node-sass@'>=3.0.0-pre <4.0.0-0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.9.4-rc1","0.2.0","0.2.1","0.2.2","0.2.3","0.2.4","0.2.5","0.2.6","0.3.0","0.4.0","0.4.1","0.4.2","0.4.3","0.4.4","0.5.0","0.5.1","0.5.2","0.5.3","0.5.4","0.6.0","0.6.1","0.6.2","0.6.3","0.6.4","0.6.5","0.6.6","0.6.7","0.7.0-alpha","0.7.0","0.8.0","0.8.1","0.8.2","0.8.3","0.8.4","0.8.5","0.8.6","0.9.0","0.9.1","0.9.2","0.9.3","0.9.4-rc1","0.9.4","0.9.5-rc1","0.9.5","0.9.6","1.0.0","1.0.1","1.0.2-alpha","1.0.2","1.0.3","1.1.0","1.1.1","1.1.2","1.1.3","1.1.4","1.2.0","1.2.1","1.2.2","1.2.3","2.0.0-beta","2.0.0","2.0.1","3.0.0-alpha.0","3.0.0-beta.2","3.0.0-beta.3","3.0.0-beta.4","2.1.0","2.1.1","3.0.0-beta.5"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! System Linux 3.13.0-24-generic
npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! cwd /home/ubuntu/.strider/data/yldio-paz-web-5526e7393fd72c3144000323
npm ERR! node -v v0.10.30
npm ERR! npm -v 1.4.21
npm ERR! code ETARGET
npm WARN prefer global jshint@2.6.3 should be installed with -g
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.0",
-    "broccoli-sass": "0.6.4",
+    "broccoli-sass": "0.6.2",
     "ember-cli": "^0.2.0",
     "ember-cli-app-version": "0.3.2",
     "ember-cli-babel": "^4.3.0",

